packages/standard-generator/src/fs.config.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
   plugins: [
     {
       modulePath: '@jpmorganchase/mosaic-plugins/SiteMapPlugin',
-      options: {}
+      options: { siteUrl: process.env.SITE_URL || 'http://localhost:3000' }
```

Comment:
does this need a doc change or read
e change ?

Reply:
We should generate a new snapshot for the Mosaic site right?  Our sitemap will be not be correct atm.

Reply:
yep I'll make sure this is explained in the docs and regen the sitemap

Although, I was having trouble with env vars not getting picked up. Where would we expect the Salt site for example to set the SITE_URL? Are they all stored in the env files or are we setting some through Vercel?

Reply:
Yeah this is becoming more of a problem, notably with disabling of the sources.  We have .env files but I think they only apply to the nextjs site. 

We may now need a more comprehensive solution that uses dotenv so that both the site and mosaic fs can read env variables...might be one for another PR tho...

Reply:
Okay cool, we are on the same page. I'll add dotenv in another PR as a prerequisite for this one.